### DISCUSSION TITLE
Which certification after completing this course

### DISCUSSION THREAD

**Discussion by manpreetkaurnew** [microsoft/ML-For-Beginners#682] [Q&A]
Hi I am a teacher helping some high school students get more info about this self paced course. we have a few students - really interested in starting this machine learning course.
my question - after they complete this, is there a microsoft certification they can set as a goal? if someone has info, pls share. Really appreciate it.

**Comment by Justmephoenix** [comment]
Sir I appreciate your work and you did a great job by helping those students . But unfortunately they are no such certification course available on Microsoft but there is a exam called DP-100 . This exam mainly consists of Data Science and ML the criteria are 
1-Designing and creating a suitable working environment for data science workloads.
2-Exploring data.
3-Training machine learning models.
4-Implementing pipelines.
5-Running jobs to prepare for production.
6-Managing, deploying, and monitoring scalable machine learning solutions.

As a candidate for this exam, you should have knowledge and experience in data science by using:

Azure Machine Learning
MLflow
  After You complete the exam you can have the certificate from Microsoft on "Designing and Implementing Data Science On Azure".
